server/user_routes.py

```
@@ -0,0 +1,192 @@
+# server/user_routes.py
+from flask import Blueprint, request, jsonify, current_app
+from flask_jwt_extended import jwt_required, get_jwt_identity, get_jwt
+from server.models import db, User
+from server.session import requires_active_session
```

Ruff (0.8.2)

`server.session.requires_active_session` imported but unused

Remove unused import: `server.session.requires_active_session`

(F401)